src/components/file-browser/file-browser-types.ts

```diff
@@ -0,0 +1,11 @@
+export interface FileBrowserFile {
```

Comment:
let's move this file over to src/libs/ajax area too (see other comment on "backends" files